analyzer/Dockerfile

```diff
@@ -0,0 +1,31 @@
+ARG RELEASE=bullseye
+FROM python:3-${RELEASE}
+ARG RELEASE
+
+RUN apt-get update -qq && apt-get upgrade -y -qq
+ADD https://raw.githubusercontent.com/LibreTime/libretime/master/tools/packages.py /tmp/packages.py
+COPY packages.ini /tmp/
+RUN apt-get install -y -qq $(python3 /tmp/packages.py --format=line ${RELEASE} /tmp/packages.ini)
```

Comment:
What about making this a single RUN, using wget instead of ADD ? Does it break caching ?

Reply:
wget will not check the cache intelligently. According to https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#leverage-build-cache, ADD will compare the checksum of the URL and the dest and invalidate the cache if they are different. However, RUN will never invalidate the cache on subsequent runs, potentially leaving the file out of date